models/brats_mri_axial_slices_generative_diffusion/docs/README.md

```diff
@@ -0,0 +1,171 @@
+2. if set `pretrained` to `True` and specifies the `perceptual_loss_model_weights_path` parameter, users are able to load weights from a local path. This is the way this bundle used to train, and the pre-trained weights are from some internal data.
+
+Please note that each user is responsible for checking the data source of the pre-trained models, the applicable licenses, and determining if suitable for the intended use.
+
+#### Example synthetic image
+An example result from inference is shown below:
+![Example synthetic image](https://developer.download.nvidia.com/assets/Clara/Images/monai_brain_image_gen_ldm2d_example_generation.png)
+
+**This is a demonstration network meant to just show the training process for this sort of network with MONAI. To achieve better performance, users need to use larger dataset like [BraTS 2021](https://www.synapse.org/#!Synapse:syn25829067/wiki/610865).**
+
+## MONAI Generative Model Dependencies
+[MONAI generative models](https://github.com/Project-MONAI/GenerativeModels) can be installed by
+```
+pip install lpips==0.1.4
+pip install git+https://github.com/Project-MONAI/GenerativeModels.git@0.2.1
+```
+
+## Data
+The training data is BraTS 2016 and 2017 from [Medical Decathlon](http://medicaldecathlon.com/). Users can find more details on the dataset (Task01_BrainTumour) at [Medical Decathlon](http://medicaldecathlon.com/).
+
+- Target: Image Generation
+- Task: Synthesis
+- Modality: MRI
+- Size: 388 3D MRI volumes (1 channel used)
+- Training data size: 38800 2D MRI axial slices (1 channel used)
+
+## Training Configuration
+If you have a GPU with less than 32G of memory, you may need to decrease the batch size when training. To do so, modify the `"train_batch_size_img"` and `"train_batch_size_slice"` parameters in the `configs/train_autoencoder.json` and `configs/train_diffusion.json` configuration files.
+- `"train_batch_size_img"` is number of 3D volumes loaded in each batch.
+- `"train_batch_size_slice"` is the number of 2D axial slices extracted from each image. The actual batch size is the product of them.
+
+### Training Configuration of Autoencoder
+The autoencoder was trained using the following configuration:
+
+- GPU: at least 32GB GPU memory
+- Actual Model Input: 240 x 240
+- AMP: False
+- Optimizer: Adam
+- Learning Rate: 5e-5
+- Loss: L1 loss, perceptual loss, KL divergence loss, adversarial  loss, GAN BCE loss
+
+#### Input
+1 channel 2D MRI Flair axial patches
+
+#### Output
+- 1 channel 2D MRI reconstructed patches
+- 1 channel mean of latent features
+- 1 channel standard deviation of latent features
+
+### Training Configuration of Diffusion Model
+The latent diffusion model was trained using the following configuration:
+
+- GPU: at least 32GB GPU memory
+- Actual Model Input: 64 x 64
+- AMP: False
+- Optimizer: Adam
+- Learning Rate: 5e-5
+- Loss: MSE loss
+
+#### Training Input
+- 1 channel noisy latent features
+- an int that indicates the time step
```

Comment:
does this bundle have time step as the input?
It's inconsistent with the `metadata.json`, may need your double confirm

Reply:
In addition, may also need to double confirm the type defined in the 3d bundle's metadata.
In readme, it says `int`, but in metadata it's `long`
https://github.com/Project-MONAI/model-zoo/pull/408/files#diff-8a35369c7144fec1b9f900c69bb3704ad52d5194181d65e3a9601c588aeaeba3L89

Reply:
The network has long int as input. In bundle training, it automatically random sample a long int. How should we put input here?

Reply:
Hi Can, I think we should use long

Reply:
Yes, I updated it